### DISCUSSION TITLE
[Bug report]: No profiles after updating to 1.5.1

### DISCUSSION THREAD

**Discussion by DanielLacambra** [iotaledger/firefly#2989] [Help & Support]
### Platform

Desktop

### Version

1.5.1

### Description

Good morning, my name is Daniel.

Yesterday I migrated to Firefly 1.5.1.
When Firefly 1.5.1 booted no profile existed.
I created a new one and retrieved it from the Ledger Nano X.
Everything was correct.
I had a balance of 10000 MIOTA and now I see 0.
I also had a balance in Shimmer and ASMB (after spending 3 months with a total balance of 15,000 MIOTA)

What could have happened?

Greetings

Daniel

### Expected behaviour

I expected to see my profiles

### Actual behaviour

No profiles in Firefly 1.5.1

### Can the issue reliably be reproduced?

No

### Operating System

Windows

### Developer Profile

No

### Profile Type

Hardware (Ledger)

### Staking

Previously staked

### Crash Status

None of the above

### Duplicate declaration

- [X] I have searched the issues tracker this issue and there is none

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by nicole-obrien** [comment]
Please can you make sure that you do not have 1.3.x version still installed on your device?

**Comment by DanielLacambra** [comment]
Hi Nicole,
I have uninstalled all Firefly versions and reinstall Firefly 1.3.3.
I continue without seeing my balance of 10000 MIOTAS.

The first think I wanted to know is ... If my 10000 MIOTAs are in a Ledger Nano X (I have two identical) there is no risk to lose my MIOTAS .. True?

I attach a file with details ..

Thanks

Daniel
[ICantRestoreMy10000MIOTAS_Wallet.docx](https://github.com/iotaledger/firefly/files/8524751/ICantRestoreMy10000MIOTAS_Wallet.docx)

**Comment by nicole-obrien** [comment]
There is no risk of loosing them as they are store on the Tangle, and you have the private keys stored in the ledger. Firefly is just a means to view and manage these IOTA.

Please make sure you install 1.5.1 and do not have any other version installed on your device.

**Comment by DanielLacambra** [comment]
Hi Nicole, I have partially solved the problem. But I think there is a bug in the recovery process from a Ledger Nano X.

When I migrated to Firefly 1.5.1 (from 1.3) I had two ledger Nano X wallets :  10000 Miotas and 0 Miotas (with the corresponding SMR and ASMB tokens)
My Firefly profile disappeared so I had to recover it from ledger (I have only Firefly 1.5.1 version on my computer)

Only 1 Wallet was recovered ==> The wallet with 0 MIOTAS  ... **Why?   I expected the two wallets to be recovered**

I tried to create manually a new Wallet (in the same profile) but Firefly told me that first of all I needed to use the Wallet1 (with 0 MIOTAS).... At the end I decided to transfer 60 MIOTAS from Binance to Wallet1 (just to do some tests)... Then I was able to create the Wallet2 .... Suddenly the 10.000 MIOTAS appeared in this Wallet ...!!!!!!!!!!!!!!!!!!!!!!!

The other question is ???? Where are the SMR and ASMB tokens that I got after 90 staking days?

Thanks

**Reply by nicole-obrien** [reply]
Hi Daniel, please can you use the balance finder in the ` settings > advanced > balance finder`

**Reply by DanielLacambra** [reply]
Hi Nicole, I executed Balance Finder 5 times but the SMR and ASMB tokens have not appeared.

What can I do?


Thanks

**Reply by DanielLacambra** [reply]
HI Nicole,

I can ping raw.githubusercontent.com. 
No problems with my hosts file. It's empty.
5 minutes ago I tried to Find Balances but the ASMB and SMR tokens have not appeared...

What can I do?

**Reply by DanielLacambra** [reply]
Hi Nicole,

what you can do to recover the SMR and ASMB tokens?
They still don't appear in Firefly's wallet and I don't want to lose them.

Remember that the problem was caused by a Firefly update that deleted my profiles



Thanks

Daniel

**Comment by Alfext** [comment]
I've got similar issue overhere. I have updated firefly to 1.51. I've no problem to see my Miota balance, but all my SMR an ASMB tokens have dissapeared. I've also executed Blance Finder more than 25 times, but my SMR and ASMB tokens have not appeared yet.

**Reply by MiKeF05** [reply]
The most likely reason that your SMR and ASMB balance is not displayed is that you can't connect to github. The necessary info is fetched from github. You can check your staking address on thetangle.org. You should see your rewards from the first staking period. There will be an alternative way to fetch the info in the next release.

**Reply by Alfext** [reply]
Thanks MikeF05!! Your answer was the key to solve my problem. 
Asking other people smarter than me, we have noticed that in my case, the problem was caused by a filter in the "host" file of my Windows.
Checking that file, we could see that the acces fron my computer to the adress "[raw.githubusercontent.com](http://raw.githubusercontent.com/)" was blocked.
As soon as I eliminated that block un my host file, my ASMB and SMR reappeared in my FireFly.
We can supose that this block could exist not only in the host file of Windows, but also in an adblock that anyone could have installed, or could even be blocked on a DNS level.
The problem is solved in my case

**Reply by armentazz** [reply]
Hello: I have seen this comment as a response to several similar problems, but I have not seen any comments from anyone else that have worked for them.
I have several days looking for the answer and I'm already desperate. I restored my Firefly from seed (I don't have backup addresses) but my Iota balance is 0, I only see first period rewards.

**Comment by DanielLacambra** [comment]
Thanks to both of you. I verified I can access
http://raw.githubusercontent.com/. I searched for balances (in Firefly)
again and again but the ASMB and SMR tokens did not appear...

I hope someone from Iota can help me.

Can I see the SMR and ASMB tokens in thetangle.org... how should I do it?

Thanks

Daniel

**Reply by Alfext** [reply]
Díd you try ping raw.githubusercontent.com? What was the result?

**Reply by DanielLacambra** [reply]
Yes. I can ping raw.githubusercontent.com. 
No problems with my hosts file. It's empty.
5 minutes ago I tried to Find Balances but the ASMB and SMR tokens have not appeared...

**Comment by DanielLacambra** [comment]
Hello, you can try to create the wallet manually and then restore it from
the seed.
In my case I had two wallets (Wallet 1 with 0 IOTAS, Wallet 2 with X Iotas).
At first I restored from Ledger Nano X but Wallet 2 was not showing up in
Firefly.
I needed to manually create Wallet2 and restore again from Ledger Nano X
==> Suddenly the balance in Wallet 2 appeared.

I hope this can help you

**Comment by pribridd** [comment]
until version 1.4.2 my smr token appeared, after updating to the next version onwards they disappeared and the smr field does not appear, only iota and asmb.
I need help to recover these tokens I don't know what happened

**Comment by DanielLacambra** [comment]
Hi everybody,

Some months ago I had the same problem when I migrate to Firefly 1.5.. I
lost SMB y ASMR tokens.

I already assumed that the smb and asmr tokens were lost.

Yesterdat I update Firefly to the Last version(1.6.x) and suddendly the
tokens appeared.


I used a LedgerNanoX.

**Reply by MiKeF05** [reply]
The tokens can't get lost because the networks are not live yet. It's a connection issue to the node. When the info can't be retrieved from the node the info regarding the token rewards is not available.

**Reply by pribridd** [reply]
and how do i solve it? about the ping I did and everything is ok.
My smr balance only appears if I go back to version 1.4.2, now the versions onwards the balance disappears, the smr field does not appear, help me

**Reply by MiKeF05** [reply]
Have you tried to update to the latest version 1.6.5?

**Reply by pribridd** [reply]
yea

**Reply by MiKeF05** [reply]
Try to use the latest version and change the node in the node settings. Then logout and back in. Additionally you can use the balance finder in the settings.

**Reply by pribridd** [reply]
I did about 60 times the balance finder and nothing.
Now I changed the no as you said and it didn't show up either

**Reply by MiKeF05** [reply]
You can also wait for the shimmer launch tomorrow (28.09.2022). You need the Firefly Shimmer wallet and then you can claim your rewards. No need to worry.

**Reply by pribridd** [reply]
thanks

**Reply by pribridd** [reply]
I downloaded firefly shimmer and my smr balance did not appear either, it continues to appear only in version 1.4.2, help me

**Comment by DanielLacambra** [comment]
Hello, you can try to create the wallet manually and then restore it from
the seed.
In my case I had two wallets (Wallet 1 with 0 IOTAS, Wallet 2 with X Iotas).
At first I restored from Ledger Nano X but Wallet 2 was not showing up in
Firefly.
I needed to manually create Wallet2 and restore again from Ledger Nano X
==> Suddenly the balance in Wallet 2 appeared.

The Smb and Asmb tokens only appeared when 3 days ago  i migrated to
Firefly 1.6.5

I hope this can help you

**Reply by pribridd** [reply]
Good Morning. I did this and updated to 1.6.5 it didn't appear, only asmb and iota appear.
In version 1.4.2 iota, asmb and smr appear. I'm too sad I stayed 90 days and nothing to solve

**Comment by DanielLacambra** [comment]
I think some day you will start Firefly and the SMB will appear. This is
what happened in my case....after 3 months or more my SMB and ASMB appeared
